src/utils.js

```diff
@@ -159,6 +160,14 @@ export function parseManifestDocument(manifest, { moduleMeta }) {
           return false;
         }
       }
+      // add missing file node for canvas_cc generated resources
```

Comment:
I'm wondering if we should leave this comment out.
But without it, we'll forget why we need to have it.

Can we move this block into an Arrow Function Expression with some good constant name that can represent this comment?

With that, we can remove the comment and make the code a bit cleaner.